src/components/SidePanel/HelpContent/helpContentMap.tsx

```diff
@@ -0,0 +1,731 @@
+                                title="Workspaces allow for a wide range of features, including:"
+                            >
+                                <BulletList
+                                    styles={styles}
+                                    items={[
+                                        <Text style={styles.textNormal}>
+                                            <Text style={styles.textBold}>Categorize</Text> and <Text style={styles.textBold}>submit</Text> expenses
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            <Text style={styles.textBold}>Approve</Text> and <Text style={styles.textBold}>reimburse</Text> expenses
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            Sync with <Text style={styles.textBold}>accounting packages</Text>
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            Connect to <Text style={styles.textBold}>company card feeds</Text>
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            Manage <Text style={styles.textBold}>Expensify Cards</Text>
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            <Text style={styles.textBold}>Chat</Text> with colleagues, partners, and clients
+                                        </Text>,
+                                        <Text style={styles.textNormal}>… and lots more!</Text>,
+                                    ]}
+                                />
+                            </ExpandableHelp>
+
+                            <ExpandableHelp
+                                styles={styles}
+                                containerStyle={styles.mb4}
+                                title="Workspaces come in two variations:"
+                            >
+                                <BulletList
+                                    styles={styles}
+                                    items={[
+                                        <Text style={styles.textNormal}>
+                                            <Text style={styles.textBold}>Collect</Text> workspaces start at $5/member, and include all the basics for running a small business.
+                                        </Text>,
+                                        <Text style={styles.textNormal}>
+                                            <Text style={styles.textBold}>Control</Text> workspaces start at $9/member, and provide advanced capabilities, more powerful accounting sync, and
```

Comment:
Out of scope for this PR, but maybe we should consider using the pricing constants instead (e.g. `SUBSCRIPTION_PRICES`) down the line

Reply:
Let's not use any CONST due to this:
> Don't use constants (eg, `:${CONST.REPORT.HELP_TYPE.POLICY_ADMINS}) because the code we generate isn't going to use them either.